pkg/registry/query.go

```diff
@@ -0,0 +1,322 @@
+package registry
+
+import (
+	"context"
+	"fmt"
+	"sort"
+
+	"github.com/operator-framework/operator-registry/internal/model"
+	"github.com/operator-framework/operator-registry/pkg/api"
+)
+
+type Querier struct {
+	pkgs model.Model
+}
+
+var _ GRPCQuery = &Querier{}
+
+func NewQuerier(packages model.Model) *Querier {
+	return &Querier{
+		pkgs: packages,
+	}
+}
+
+func (q Querier) ListPackages(_ context.Context) ([]string, error) {
```

Comment:
Are we testing these new implementations against the existing tests somewhere?

Reply:
Yes, `pkg/server/server_test.go`.

Reply:
Okay I see it (for some reason github decided to contract that file while I was reviewing it so it escaped my eye 🤦🏽 )

Reply:
Yeah GitHub has some weird behaviors like that. Super annoying.